spec/components/validated_field_component_spec.rb

```diff
@@ -28,7 +28,7 @@
   it 'renders aria-describedby to establish connection between input and error message' do
     field = rendered.at_css('input')
 
-    expect(field.attr('aria-describedby')).to start_with('validated-field-error-')
+    expect(field.attr('aria-describedby')).to include('validated-field-error-')
```

Comment:
It would be good to add some test coverage for our new behavior, like testing to see that `aria-describedby` contains the two IDs we're expecting. I think this would also help provide some better guarantees that the IDs are what we're expecting, since technically this assertion would otherwise allow `aria-describedby="definitely-not-the-correct-validated-field-error-1234"`.

```suggestion
    expect(field.attr('aria-describedby').split(' ')).to include([
      start_with('validated-field-hint-'),
      start_with('validated-field-error-'),
    ])
```

(Also incorporates https://github.com/18F/identity-idp/pull/6747#discussion_r947848626 )